zellij-server/src/panes/grid.rs

```diff
@@ -796,6 +813,14 @@ impl Grid {
             Some((self.cursor.x, self.cursor.y))
         }
     }
+
+    pub fn dump_screen(&mut self) -> String {
+        let mut lines: String = dump_screen!(self.lines_above);
```

Comment:
How about we call this `scrollback` instead of `lines`?

Reply:
Will fix this in a new commit.